libs/ledgerjs/packages/hw-app-eth/src/services/types.ts

```diff
@@ -1,3 +1,9 @@
+import { DomainServiceResolution as DomainDescriptor } from "@ledgerhq/domain-service/types";
```

Comment:
The types were all intially set in `@ledgerhq/domain-service`, but when added as a dep to LLD/LLM, we were having a cycle dependency somewhere (partially due to the `network` module of LLC I guess). So moving the types to  `@ledgerhq/types-live` was a solution to have the right types in LLM/LLD and more specifically in test files for mocks. 

I'll try to move them again in domain-service now that we moved to `axios` and see if it's possible !

I'll add a few comments along the way 👍

Reply:
Confirmed, not possible because the type `Transaction` is now using the `DomainServiceResolution`, and adding `@ledgerhq/domain-service` as a dev dep to `@ledgerhq/types-live` is creating a cycle dependency when build live-common

Reply:
This suggests that we need a dedicated `@ledger/types-domain` lib, I think that would be the cleanest pattern, but NBD for now though, just a possible improvement.

Reply:
To be honest I'm really not sure having "types" libs for everything is a nice pattern 😕

Reply:
Not everything but seems to make sense for libs that can be used in more than 1 project.
It's a solution to circular dependencies :man_shrugging: seems better than split types.
What do you think is the issue? Too many libs everywhere? Or something worse?